libdirections-onboard/src/test/java/com/mapbox/navigation/route/onboard/MainCoroutineRule.kt

```diff
@@ -3,31 +3,37 @@ package com.mapbox.navigation.route.onboard
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.test.TestCoroutineDispatcher
+import kotlinx.coroutines.test.TestCoroutineScope
 import kotlinx.coroutines.test.resetMain
 import kotlinx.coroutines.test.runBlockingTest
 import kotlinx.coroutines.test.setMain
-import org.junit.rules.TestWatcher
+import org.junit.rules.TestRule
 import org.junit.runner.Description
+import org.junit.runners.model.Statement
 
 @ExperimentalCoroutinesApi
-class MainCoroutineRule(
-    val testDispatcher: TestCoroutineDispatcher = TestCoroutineDispatcher()
-) : TestWatcher() {
+class MainCoroutineRule : TestRule {
```

Comment:
Aren't these changes the same introduced in https://github.com/mapbox/mapbox-navigation-android/pull/2409?

Reply:
Let's remove this file. @Guardiola31337 pointed it out right.